data-processing/entities/sentences/extractor.py

```diff
@@ -50,14 +58,29 @@ def parse(self, tex_path: str, tex: str) -> Iterator[Sentence]:
         # Record the current length of the plain text so account for the extractor bug
         length_so_far_in_plain_text = 0
         for i, sentence in enumerate(segmenter.segment(plaintext)):
-            # Since the sentence extractor has several bugs related to finding start and end indicies
-            # we will simply set them ourselves.
+            # The pysbd module has several open bugs and issues which are addressed below.
+            # As of 3/23/20 we know the module will fail in the following ways:
+            # 1. pysbd will not break up the sentence when it starts with a punctuation mark or space.
+            #    ex: ". hello. world. hi."
+            #    sol: check for sentences being longer than 1000 characters.
+            # 2. pysbd indexes are sometimes incorrectly set
+            #    ex: "hello. world. 1) item one. 2) item two. 3) item three" or "hello!!! world."
+            #    sol: set indexes manually using string search + sentence length
+            # 3. pysbd uses reserved characters for splitting sentences
+            #    ex: see PSYDB_RESERVED_CHARACTERS list.
+            #    sol: throw a warning if the sentence contains any of these characters.
+            if len(sentence.sent) > 1000:
+                logging.warning('exceptionally long sentence, this might indicate the sentence extractor is broken')
```

Comment:
```suggestion
                logging.warning('Exceptionally long sentence (length %d), this might indicate the sentence extractor failed to properly split text into sentences.', len(sentence.sent))
```

Reply:
Just making some small updates to the logging---capitalizing it, and trying to clarify the log message.